akka-docs/src/main/paradox/typed/fault-tolerance.md

```diff
@@ -74,6 +83,39 @@ Java
 
 Each returned behavior will be re-wrapped automatically with the supervisor.
 
+## Child actors remain when restarting
+
+Child actors are not influenced when the parent actor is restarted. The restarted parent instance will have the same
+children as before the failure.
```

Comment:
👍 as discussed in a call, this is good, and less confusing than the state of untyped, indeed good to only document it.